rust-runtime/aws-smithy-http-server/src/rejection.rs

```diff
@@ -227,3 +227,9 @@ impl From<serde_urlencoded::de::Error> for SmithyRejection {
         SmithyRejection::Deserialize(Deserialize::from_err(err))
     }
 }
+
+impl From<nom::Err<nom::error::Error<&str>>> for SmithyRejection {
+    fn from(_err: nom::Err<nom::error::Error<&str>>) -> Self {
+        SmithyRejection::MissingQueryString(MissingQueryString)
```

Comment:
`MissingQueryString` is the wrong rejection type, since we're not looking in the query string, but the URI path. `SmithyRejection::Deserialize` should be fine for now.